File: 프로그래머스/0/181837. 커피 심부름/커피 심부름.py
# 아메리카노 4500 라테 5000

def solution(order):
    answer = 0
    for coffee in order:
        if 'americano' in coffee or 'anything' in coffee:
            answer += 4500
        elif 'cafelatte' in coffee:
            answer += 5000
    return answer

File: 프로그래머스/0/181837. 커피 심부름/커피 심부름.java
class Solution {
    public int solution(String[] order) {
        int answer = 0;
        for (String cafe : order) {
            if (cafe.contains("americano") || "anything".equals(cafe)) {
                answer += 4500;
            } else {
                answer += 5000;
            }
        }
        
        return answer;
    }
}